packages/medusa/src/api/routes/admin/payment-collections/capture-payment.ts

```diff
@@ -0,0 +1,13 @@
+import { PaymentCollectionService } from "../../../../services"
```

Comment:
thought(non-blocking): I think we should just kill these and unify capturing payments so it only happens through the PaymentsService.